unit_tests/test_setups.py

```diff
@@ -281,6 +281,99 @@ def test_setup_keck_deimos_multiconfig_clean():
     assert len(ps.fitstbl) == 23, 'Incorrect number of table rows.'
 
 
+def test_setup_keck_mosfire():
```

Comment:
is this really part of this PR?

Reply:
I definitely did not write this, but perhaps something did not get merged into develop when I pulled it in.